debian/bionic/foreman/foreman-plugin.mk

```diff
@@ -11,7 +11,7 @@ build:
 	)
 	GEM_PATH=$$(cd /usr/share/foreman && /usr/bin/foreman-ruby /usr/bin/bundle show $(PLUGIN)) && \
 	cp -rp $${GEM_PATH}/public/assets ./ && \
```

Comment:
At this point, does `&& \` make sense?

Reply:
the `\` is needed, to get `$GEM_PATH`, otherwise it's a new shell… the `&&` not so much

Reply:
Ah, so it's combining them and `&&` is just the way to run multiple commands in the same shell. Got it.

Reply:
yeah, running subshells in make is wild